webapp/channels/src/components/sidebar/add_channel_dropdown.tsx

```diff
@@ -135,7 +143,13 @@ const AddChannelDropdown = ({
         );
     };
 
+    const dispatch = useDispatch<DispatchFunc>();
+
     const trackOpen = (opened: boolean) => {
+        dispatch(openModal({
```

Comment:
I may be missing how the logic is supposed to work, so... could you explain why we open the modal in `add_channel_dropdown` component when we call `trackOpen`?

Reply:
As has traditionally been the reason for such weird diffs, it was left over from debugging.